soccer/src/rj_vision_filter/kick/estimator/ChipKickEstimator.cpp

```diff
@@ -0,0 +1 @@
+#include <rj_vision_filter/kick/estimator/ChipKickEstimator.hpp>
```

Comment:
idk why it shows as this being a new file instead of being moved + having a diff.